client/src/bug-report.js

```diff
@@ -379,6 +379,10 @@ customElements.define(
       shadowRoot
         .getElementById("submit")
         .addEventListener("click", async function () {
+          const submitButton = shadowRoot.getElementById("submit");
```

Comment:
Is there a better way to get access to the submit button, considering this is a callback on the same element. Please explore it

Reply:
We can try Debouncing function and use a timeout for allowing multiple clicks.

The second way is to Loading Indicator to inform the user that the submission is in progress. We can create a div with innertext set as 'Submission in progress..' while we disable the submit button for multiple clicks. Before re-enabling the button, we remove this div element.

In both the methods, we have to disable the submit button, which we are doing anyway in the current commit.